src/components/Check/Check.scss

```diff
@@ -7,28 +7,35 @@
 // --------------------------------------------------
 // Check styles
 
-$checkBoxHeight: 20px;
+$checkBoxHeight: 22px;
 
 .ms-Check {
-  display: inline-block;
-  cursor: default;
-  vertical-align: top;
+  font-family: FabricMDL2Icons;
```

Comment:
Updated.  Didn't need it at line 13,  removed it from line 62 and added ms-icon class to the element instead.